app/services/album.js

```diff
@@ -0,0 +1,6 @@
+const axios = require('axios');
+
+exports.albums = () =>
+  axios.get('https://jsonplaceholder.typicode.com/albums').then(response => {
+    console.log(response);
```

Comment:
Don't use console.log, use the logger. You can use it by requiring the app/logger folder like so
```
const logger = require(../logger);
...
logger.info(`Received albums: ${JSON.stringify(response)}`)
```

You can also use other logging levels, like logger.error. Use accordingly.

Reply:
You are also missing a response here. Since the function inside the `.then` opens brackets, it does not automatically return anything other than `undefined`. You have to manually add the `return` statement for the `response`.

Reply:
Also, what happens if something fails? You need to add a `.catch` to handle possible errors with comunication. We usually wrap the caugth error inside an errors construct defined in the `app/errors.js` file and then map said construct to a status code inside the `app/middleware/errors.js` middleware